Modules/_sysconfig.c

```diff
@@ -0,0 +1,91 @@
+// _sysconfig provides data for the Python sysconfig module
+
+#ifndef Py_BUILD_CORE_BUILTIN
+#  define Py_BUILD_CORE_MODULE 1
+#endif
+
+#include "Python.h"
+
+#include "pycore_importdl.h"   // _PyImport_DynLoadFiletab
+
+
+/*[clinic input]
+module _sysconfig
+[clinic start generated code]*/
+/*[clinic end generated code: output=da39a3ee5e6b4b0d input=0a7c02d3e212ac97]*/
+
+#include "clinic/_sysconfig.c.h"
+
+static int
+add_string_value(PyObject *dict, const char *key, const char *str_value)
+{
+    PyObject *value = PyUnicode_FromString(str_value);
+    if (value == NULL) {
+        return -1;
+    }
+    int err = PyDict_SetItemString(dict, key, value);
+    Py_DECREF(value);
+    return err;
+}
+
+/*[clinic input]
+_sysconfig.config_vars
+
+Returns a dictionary containing variables intended to be exposed by sysconfig.
+[clinic start generated code]*/
+
+static PyObject *
+_sysconfig_config_vars_impl(PyObject *module)
+/*[clinic end generated code: output=9c41cdee63ea9487 input=1458713a8d30b474]*/
+{
+    PyObject *config = PyDict_New();
+    if (config == NULL) {
+        return NULL;
+    }
+    if (_PyImport_DynLoadFiletab[0]) {
+        if (add_string_value(config, "EXT_SUFFIX", _PyImport_DynLoadFiletab[0]) < 0) {
+            Py_DECREF(config);
+            return NULL;
+        }
+    }
```

Comment:
This is one of the things I wanted to avoid :sweat_smile:

I think it's probably okay, but one thing to keep in mind is that `EXT_SUFFIX` is usually used as the extension to give native modules. We likely want to make sure we are setting it to the one that includes `SOABI`, not a different one (eg. the `abi3` one). In this case, I think we should probably be okay since we know Windows will always have one, and put it in the first entry.